### DISCUSSION TITLE
Water meter reading doesn't change

### DISCUSSION THREAD

**Discussion by yaydogan** [bemasher/rtlamr#183] [Q&A]
Hi,

I am using rtlamr to scan for 60 secs and sleep for 10 mins. I can read my electric and gas meters with SCM message type and consumption values  are updating every 10 mins. I am reading my water meter (schlumberger) with R900 message type but the consumption value is  not changing. I tried  R900 and R900BCD message types: reading is not changing and the consumption value is different for the same meter ID.

When my meter had 208370, the readings with R900 and R900BCD reported these values:
![image](https://user-images.githubusercontent.com/16845183/111402546-c2bdb000-86a1-11eb-9005-24d31bcf5cf5.png)
![image](https://user-images.githubusercontent.com/16845183/111402575-cf420880-86a1-11eb-88e4-abb95d3baa45.png)

Value I read with R900BCD is close to actual reading. Is the last 2 digits on the meter fractions and not being reported in the consumption? Is that how I am supposed to interpret the consumption field?

Thanks in advance.

**Comment by bemasher** [comment]
Typically meters transmit higher precision than they show on any displays or dials.

**Reply by zachliu** [reply]
Sadly this is not the case for these new [Dresser series D gas meters](https://dresserutility.com/wp-content/uploads/NGS-MI-0039.pdf). The measuring part is also digital and controlled by an internal software, which transfers data to the Itron 100G DLS data logger, which then transmits on RF. While Itron still transmits every minute, the internal software only sends data every 100 cubic feet (1 CCF). I'll never have "real-time" gas usage monitoring on this meter :cry:
